ntc_templates/templates/ubiquiti_edgerouter_show_ipv6_route.textfsm

```diff
@@ -0,0 +1,16 @@
+Value Filldown CODE (\w{1,2})
```

Comment:
The test file does not include an example where Filldown would be needed. Is this needed, and the test case needs to be updated, or is this not needed?

Reply:
The filldown is indeed not necessary. I have removed it.